src/database/models/callDescription.js

```diff
@@ -0,0 +1,49 @@
+module.exports = (sequelize, Sequelize) => {
+    return sequelize.define('callDescription', {
+        callDescription_id: {
```

Comment:
How can you mic camelCase_and_snake_case?
Please change all of the field names to snake_case.